managed/devops/bin/yb_platform_util.sh

```diff
@@ -0,0 +1,525 @@
+# Copyright (c) YugaByte, Inc.
+#
+# Copyright 2019 YugaByte, Inc. and Contributors
+#
+# Licensed under the Polyform Free Trial License 1.0.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+# https://github.com/YugaByte/yugabyte-db/blob/master/licenses/POLYFORM-FREE-TRIAL-LICENSE-1.0.0.txt
+
+
+# Save user current directory and move to shell script folder to run shell script
+
+bold="\033[1m"
+normal="\033[0m"
+
+
+function ProgressBar {
+# Process data
+    let _progress=(${1}*100/${2}*100)/100
+    let _done=(${_progress}*4)/10
+    let _left=40-$_done
+# Build progressbar string lengths
+    _fill=$(printf "%${_done}s")
+    _empty=$(printf "%${_left}s")
+
+printf "\rProgress : [${_fill// /#}${_empty// /-}] ${_progress}%%\n"
+
+}
+
+function TaskStatus {
+    task_id=$1
+    base_url=$(echo "$YB_PLATFORM_URL" | sed 's/\/$//')
+    auth_uuid="$YB_PLATFORM_API_TOKEN"
+    customer=$2
+    success_message=$3
+    _end=100
+     python_3="False"
+    if [[ "$(python3 -V)" =~ "Python 3" ]]
+    then
+        python_3="True"
+    fi
+
+    while :
+    do
+
+        if [[ $python_3 == "True" ]]
+        then
+            percentage_response=$(python3 -c "import yb_platform_util_py3; \
```

Comment:
Good idea,
Let me check, while fixing those comments I feel it would be better if I can combine python2 and python3 file. i am checking that. That will fix this issue probably.